server/block/sandstone_type.go

```diff
@@ -71,6 +71,26 @@ func (s sandstone) String() string {
 	panic("unknown sandstone type")
 }
 
+// Smooth varient
+func (s sandstone) Smooth() bool {
+	return s == 3
+}
+
+// Cut varient
+func (s sandstone) Cut() bool {
+	return s == 1
+}
```

Comment:
You should be able to compare `SandstoneType`s directly using `==`, so I think we don't need these functions. Could just replace them with `s == SmoothSandstone()` and `s == CutSandstone()` respectively.

Reply:
I got an error on the conversion to use the type:
`invalid operation: s == SmoothSandstone() (mismatched types sandstone and SandstoneType)`
please let me know you preferred option.

Reply:
Sorry, I wasn't clear enough with what I meant. I quickly made a change and it should be all good now!